### PR TITLE
image generation scripts

### PR DESCRIPTION
**Pull request by busmangit**
octave scripts to generate benchmarking images

### CODE REVIEW

File: ptrack-gentest/cleanCloseObjectCluster.m [hunk 1]
```diff
@@ -0,0 +1,57 @@
+function [xcheck,ycheck,Nfinal]=cleanCloseObjectCluster(x,y, cx, cy, D, L, M)
```
Comment (juanssl) [inline ptrack-gentest/cleanCloseObjectCluster.m:1]: Function is missing comments, input/output data types and format.

File: ptrack-gentest/gentests.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+C=$#
```
Comment (juanssl) [inline ptrack-gentest/gentests.sh:2]: Script is missing
#!/bin/bash or #!/bin/sh

File: ptrack-gentest/readme.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+requires octave to run (install with apt-get install octave in ubuntu)
```
Comment (juanssl) [inline ptrack-gentest/readme.txt]: Better re-write the file as Markdown.
Use standard name convention as "README.md"

File: ptrack-gentest/cleanCloseObjectCluster.m (hunk 1 above)
Comment (busmangit) [reply ptrack-gentest/cleanCloseObjectCluster.m:1]: documentation added, no data types as matlab/octave does no requires data types

File: ptrack-gentest/gentests.sh (hunk 2 above)
Comment (busmangit) [reply ptrack-gentest/gentests.sh:2]: corrected

File: ptrack-gentest/readme.txt (hunk 3 above)
Comment (busmangit) [reply ptrack-gentest/readme.txt]: corrected

File: ptrack-gentest/cleanCloseObjectCluster.m (hunk 1 above)
Comment (juanssl) [reply ptrack-gentest/cleanCloseObjectCluster.m:1]: Documentation shouldn't be at the top of the function?

File: ptrack-gentest/cleanCloseObjectCluster.m~ [hunk 4]
```diff
@@ -0,0 +1,69 @@
+function [xcheck,ycheck,Nfinal]=cleanCloseObjectCluster(x,y, cx, cy, D, L, M)
```
Comment (juanssl) [inline ptrack-gentest/cleanCloseObjectCluster.m~:1]: Unwanted file .m~

**Review by juanssl — COMMENTED** [review]
Is this binary file "octave-workspace" required or is it autogenerated...
Binary files should be tracked using Git LFS

File: ptrack-gentest/cleanCloseObjectCluster.m (hunk 1 above)
Comment (busmangit) [reply ptrack-gentest/cleanCloseObjectCluster.m:1]: no, it is matlab style docs

File: ptrack-gentest/cleanCloseObjectCluster.m~ (hunk 4 above)
Comment (busmangit) [reply ptrack-gentest/cleanCloseObjectCluster.m~:1]: ok deleted, it is a tmp file created by the editor

**Pull request closed without merging**